packages/docusaurus-theme-classic/src/theme/IconEdit/index.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {Props} from '@theme/IconArrow';
+
+const IconArrow = (props: Props): JSX.Element => {
+  return (
+    <svg
+      fill="currentColor"
+      height="1.2em"
+      width="1.2em"
+      preserveAspectRatio="xMidYMid meet"
+      role="img"
+      viewBox="0 0 40 40"
+      style={{
+        marginRight: '0.3em',
+        verticalAlign: 'sub',
+      }}
```

Comment:
"external" styles should rather be passed as props and keep in the parent component.

Some refs:
https://visly.app/blogposts/layout-isolated-components
https://mxstbr.com/thoughts/margin